crates/metadata/compatibility_list.json

```diff
@@ -8,6 +8,9 @@
         },
         "4.0.0-rc.2": {
             "ink": ["5.0.0-alpha", "5.0.0"]
+        },
+        "4.0.0-rc.3" : {
+            "ink": ["5.0.0-rc.2", "5.0.0"]
```

Comment:
we have overlapping range with previous entry. As I remember in 5.0.0-rc
 we have changed storage keys, so probably we should have such configuration:
        "4.0.0-rc.1": {
            "ink": [">=5.0.0-alpha, <5.0.0-rc.1"]
         "4.0.0-rc.3" : {
            "ink": ["5.0.0-rc.1", "5.0.0"]
           }

Reply:
Now we do not have any c-c version compatible with `5.0.0-rc.1`